### DISCUSSION TITLE
support for open modal window?

### DISCUSSION THREAD

**Discussion by vincentff7** [grafana/grafana#50777] [Dashboards, Panels & Visualizations]
support for open modal window instead of open in new browser tab?

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
